### PR TITLE
Update individual playbooks from Retry Installation

### PR DESCRIPTION
**Pull request by danielfn**

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@danielfn Thank you for this! I assume this applies to OCP 3.11?
@gpei Can you please provide QE review?

**Comment by danielfn** [comment]
@ahardin-rh yes, I have checked this against OKD 3.11

**Comment by gpei** [comment]
@ahardin-rh for `Admin Console Install`, @yapei should know more

**Comment by ahardin-rh** [comment]
@yapei PTAL

File: install/running_install.adoc [hunk 1]
```diff
@@ -483,15 +483,15 @@ The following table lists the playbooks in the order that they must run:
 |Web Console Install
 |*_{pb-prefix}playbooks/openshift-web-console/config.yml_*
 
+|Admin Console Install
+|*_{pb-prefix}playbooks/openshift-console/config.yml_*
+
```
Comment (yapei) [inline install/running_install.adoc]: Admin Console Install looks good to me, sorry for late reply

Comment (ahardin-rh) [reply install/running_install.adoc]: Thank you!

**Pull request merged**